### DISCUSSION TITLE
Cleanup old tasks

### DISCUSSION THREAD

**Discussion by UNiXMIT** [semaphoreui/semaphore#2303] [General]
Is there a way to clean up old tasks? Perhaps configure it to only keep the last n number of tasks?

**Comment by UNiXMIT** [comment]
I see this feature was added in v2.10.18 
Thank you

**Reply by dwedia** [reply]
> v2.10.18

Did you find a way to do this, to an existing SemaphoreUI instance?

I have tried to set ' "max_tasks_per_template": 1 ' in my config.json file, but it just it doesnt seem to do anything in particular...

**Reply by fiftin** [reply]
`max_tasks_per_template` works in v2.12.8.
